### ISSUE TITLE
Minimum GPU requirements for training and evaluation?

### ISSUE THREAD

**Issue by eronez** [ayushjain1144/odin#4]
I noticed in the paper it mentioned training with **2 NVIDIA A100s with 40GB VRAM**.
I would like to know the **minimum GPU requirements** for training or evaluation. 
For example, can I reproduce the results using a GPU with 16GB VRAM?

At last,is there anyone replicated this wonderful work? Could you share what configurations you used?

**Comment by ayushjain1144** [comment]
Hi, I think you would need atleast 30GB RAM for either training or evaluation.

16GB GPU could work too but you would need to add gradient checkpointing and tricks like that. 

Ofcourse, I haven't tried it myself with 16GB GPU, so you might need to play around with the hyperparameters. But I have used 30GB GPUs during the development of ODIN, and didn't need to change hyperparameters (despite using smaller batch size)

**Comment by eronez** [comment]
> Hi, I think you would need atleast 30GB RAM for either training or evaluation.
> 
> 16GB GPU could work too but you would need to add gradient checkpointing and tricks like that.
> 
> Ofcourse, I haven't tried it myself with 16GB GPU, so you might need to play around with the hyperparameters. But I have used 30GB GPUs during the development of ODIN, and didn't need to change hyperparameters (despite using smaller batch size)

Thank you very much.
